app/screens/modal/NoInternetConnection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '../../basicComponents';
+import { ExternalLinks } from '../../../shared/constants';
+import Modal from '../../components/common/Modal';
+import ReactPortal from './ReactPortal';
+
+const ButtonsWrapper = styled.div<{ hasSingleButton?: boolean }>`
+  display: flex;
+  flex-direction: row;
+  justify-content: ${({ hasSingleButton }) =>
+    hasSingleButton ? 'center' : 'space-between'};
+  margin: auto 0 15px 0;
+  padding-top: 30px;
+`;
+
+const ErrorMessage = styled.pre`
+  font-size: 14px;
+  line-height: 16px;
+  word-wrap: break-word;
+  white-space: pre-wrap;
+  flex: 1;
+  overflow-y: auto;
+`;
+
+const getOnLineStatus = () =>
+  typeof navigator !== 'undefined' && typeof navigator.onLine === 'boolean'
+    ? navigator.onLine
+    : true;
+
+const useNavigatorOnLine = () => {
+  const [status, setStatus] = useState(getOnLineStatus());
+
+  useEffect(() => {
+    function setOnline() {
+      setStatus(true);
+    }
+    function setOffline() {
+      setStatus(false);
+    }
+    window.addEventListener('online', setOnline);
+    window.addEventListener('offline', setOffline);
+
+    return () => {
+      window.removeEventListener('online', setOnline);
+      window.removeEventListener('offline', setOffline);
+    };
+  }, [setStatus]);
+
+  return status;
+};
+
+const NoInternetConnection = () => {
+  const navigateToDiscord = () => window.open(ExternalLinks.Discord);
+  const [isIgnore, setIgnore] = useState(false);
+  const isOnline = useNavigatorOnLine();
+
+  if (isOnline || isIgnore) {
+    return null;
+  }
+
+  return (
+    <ReactPortal modalId="no-internet-connection">
+      <Modal header="NO INTERNET CONNECTION" width={600} height={350}>
+        <ErrorMessage>
+          Please, check your internet connection. Once you&apos;ll get connected
+          this dialog should disappear automatically.
+          {'\n'}
+          However, you can unlock the wallet even without an internet
+          connection. Then click the &quot;Ignore&quot; button and unlock the
+          wallet as usual.
```

Comment:
```suggestion
          {'\n'}
          Please, check your Internet connection. 
          Once reconnected, this window will disappear automatically.
          {'\n'}
          {'\n'}
          You can still log in to your wallet being offline. 
          {'\n'}
          Just click &quot;Ignore&quot; and proceed.
```


I think the original version is too long/too complicated and leaves some kind of a feeling that we could use our wallet _as usual_, I would avoid expectations on that... 

<img width="701" alt="Screenshot 2023-02-17 at 23 18 06" src="https://user-images.githubusercontent.com/89876259/219811515-a072f9a2-1659-4419-a4b9-3bad0ccfedc7.png">

VS

<img width="701" alt="Screenshot 2023-02-17 at 23 45 05" src="https://user-images.githubusercontent.com/89876259/219811659-14f66efe-d061-42bb-bc60-8027634d1809.png">

Reply:
wdyt? @brusherru  @maparr

Reply:
@monikasmolarek , more clear, will replace.